lib/customer.rb

```diff
@@ -0,0 +1,31 @@
+require "csv"
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  def self.all
+    all_customer_info = []
+    CSV.open("/Users/angelaoh/documents/grocery-store/data/customers.csv", "r").each do |info|
```

Comment:
Oh no! What you'll want here instead is `data/customers.csv`, which is a path that running `rake` on any machine will understand